recipes/python-toolchain/deactivate.bat

```diff
@@ -0,0 +1 @@
+set "PIP_CONFIG_FILE="
```

Comment:
Haven't seen quotes on unset (there are no spaces in this, so it should not be needed), but it will probably also not hurt :-)